<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cálculo de la Hipotenusa (Teorema de Pitágoras)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-top-color: rgb(181, 255, 164);
background-color: rgb(181, 255, 164);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_instruccion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string>Introduce los catetos:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_cateto1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 8pt &quot;Segoe Script&quot;;
border-radius: 10px;

</string>
    </property>
    <property name="placeholderText">
     <string>Cateto 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_resultado">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_cateto2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 8pt &quot;Segoe Script&quot;;
border-radius: 10px;
</string>
    </property>
    <property name="placeholderText">
     <string>Cateto 2</string>
    </property>
   </widget>
   <widget class="QSlider" name="Slider_cateto1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QSlider::groove:horizontal {
            border: 1px solid #999999;
            height: 8px;
            background: white;
            margin: 2px 0;
        }
        QSlider::handle:horizontal {
            background: rgb(0, 170, 0);
            border: 1px solid #5c5c5c;
            width: 18px;
            margin: -2px 0;
            border-radius: 3px;
        }</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider_cateto2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QSlider::groove:horizontal {
            border: 1px solid #999999;
            height: 8px;
            background: white;
            margin: 2px 0;
        }
        QSlider::handle:horizontal {
            background: rgb(0, 170, 0);
            border: 1px solid #5c5c5c;
            width: 18px;
            margin: -2px 0;
            border-radius: 3px;
        }</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_resultado_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string>Resultado:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
